packages/user/src/services/user-module.ts

```diff
@@ -141,6 +141,25 @@ export default class UserModuleService<
     return Array.isArray(data) ? serializedInvites : serializedInvites[0]
   }
 
+  @InjectTransactionManager("baseRepository_")
+  private async createInvites_(
+    data: UserTypes.CreateInviteDTO[],
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<TInvite[]> {
+    // expiration date in 10 days
+    const expirationDate = new Date().setDate(new Date().getDate() + 10)
```

Comment:
question: Should this be configurable?

Reply:
we could make it configurable with options, I'll include that when looking at the invite flow instead of here so this can land on dev asap?

Reply:
Let's do it 👍